packages/firestore/src/local/shared_client_state.ts

```diff
@@ -840,6 +861,15 @@ export class WebStorageSharedClientState implements SharedClientState {
               return this.handleOnlineStateEvent(onlineState);
             }
           }
+        } else if (event.key === this.sequenceNumberKey) {
+          if (this.sequenceNumberHandler) {
```

Comment:
If I am understanding this correctly, this should always be set at this point. Can we change this to an assert?

Reply:
Done.